backend/spherre/tests/test_service/test_transaction_service.py

```
@@ -0,0 +1,352 @@
+            TransactionService.create_transaction(
+                transaction_id=2,
+                account=self.account,
+                status=TransactionStatus.INITIATED,
+                type=TransactionType.TOKEN_SEND,
+                proposer=non_member,
+                date_proposed=self.current_time
+            )
+            
+    def test_approve_transaction(self):
+        # Create a test transaction
+        transaction = TransactionService.create_transaction(
```

Ruff (0.11.9)

Local variable `transaction` is assigned to but never used

Remove assignment to unused variable `transaction`

(F841)